### DISCUSSION TITLE
Problem installing tidyverse on raven Rstudio server

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1235] [General]
Trying to get tidyverse installed on my account I get

```
ERROR: dependencies ‘ggplot2’, ‘httr’, ‘rvest’, ‘xml2’ are not available for package ‘tidyverse’
* removing ‘/home/sr320/R/x86_64-pc-linux-gnu-library/4.0/tidyverse’
```
I then go down the dependencies and continue to get `not available`.

**Comment by kubu4** [comment]
Installing now (it's taking awhile), but haven't encountered any errors yet. Did you error message pop up immediately?

I'll report back once install is complete. Then, we'll have to see if the package install is available to all users...

**Reply by kubu4** [reply]
Actually, there are some errors popping up, but I think I've seen some of them before. Has to do with missing system packages in Ubuntu. Once the `tidyverse` installation process completes, I'll work on getting those other system packages installed...

**Reply by kubu4** [reply]
Okay, install is complete. I've installed the various Ubuntu system packages that were needed and then installed the R dependencies needed by `tidyverse`.

Login to your account and see if you can load `tidyverse` (you may need to quit your current R session and start a new one).

If the `tidyverse` packages aren't available, then you'll need to try installing again.

**Reply by sr320** [reply]
Worked! thanks!
